packages/api-client/playground/app/main.ts

```diff
@@ -1,5 +1,5 @@
 import { createApiClientApp } from '@/layouts/App'
 
 createApiClientApp(document.getElementById('scalar-client'), {
-  proxyUrl: 'https://proxy.scalar.com',
+  proxyUrl: 'http://localhost:5051',
```

Comment:
this should most likely still be the proxy for deploying correct?

Reply:
omg, that shouldn’t be in here